setup.py

```diff
@@ -21,15 +21,9 @@
     author="ESSS",
     author_email='foss@esss.co',
     classifiers=[
-        'Development Status :: 2 - Pre-Alpha',

```

Comment:
I think we should advertise it as `'Development Status :: 3 - Alpha'`

Reply:
- [x] Add 'Development Status :: 3 - Alpha`